docs/docs.scss

```diff
@@ -19,6 +19,12 @@ $snippet-border: 2px solid #CCC;
 $icon-block-bg: #eee;
 $icon-block-size: 75px;
 
+.daptiv-grid{
+    .box-column {
+        border: 1px solid rgba(86, 61, 124, .15);
```

Comment:
:+1: ah! thanks i didn't notice it was docs only. still, should use a color from our palette, promote best practices :sparkles: